scss/tech.scss

```diff
@@ -1,3 +1,53 @@
+@import 'colors';
+@import 'fonts';
+
+.tech-container {
+  background-color: $earls-green;
+  border-radius: 5px;
+  height: auto;
```

Comment:
I think this is default prop, you shouldnt need this

Reply:
You are right!